index.js

```diff
@@ -1,88 +1,116 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+let counter = 1
+counter = 2
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
+const name = "shmuly"
 // *** after testing, comment the line causing the error back in 
```

Comment:
Nice work using the "let" and "const" keywords in the correct context when defining JS variables.